Cubical/Data/FinSet/Properties.agda

```diff
@@ -101,51 +98,55 @@ isPropIsFinSet' {A = A} (n , equivn) (m , equivm) =
     n≡m : n ≡ m
     n≡m = rec (isSetℕ n m) (λ p → p) ∥n≡m∥
 
-isFinSet≡isFinSet' : isFinSet A ≡ isFinSet' A
-isFinSet≡isFinSet' {A = A} = hPropExt isPropIsFinSet isPropIsFinSet' to from
-  where
-    to : isFinSet A → isFinSet' A
-    to ∣ n , equiv ∣ = n , ∣ equiv ∣
-    to (squash p q i) = isPropIsFinSet' (to p) (to q) i
+-- logical equivalence of two definitions
 
-    from : isFinSet' A → isFinSet A
-    from (n , ∣ isFinSet-A ∣) = ∣ n , isFinSet-A ∣
-    from (n , squash p q i) = isPropIsFinSet (from (n , p)) (from (n , q)) i
+isFinSet→isFinSet' : isFinSet A → isFinSet' A
+isFinSet→isFinSet' ∣ n , equiv ∣ = n , ∣ equiv ∣
+isFinSet→isFinSet' (squash p q i) = isPropIsFinSet' (isFinSet→isFinSet' p) (isFinSet→isFinSet' q) i
 
-FinSet≡FinSet' : FinSet ℓ ≡ FinSet' ℓ
-FinSet≡FinSet' = ua (isoToEquiv (iso to from to-from from-to))
-  where
-    to : FinSet ℓ → FinSet' ℓ
-    to (A , isFinSetA) = A , transport isFinSet≡isFinSet' isFinSetA
+isFinSet'→isFinSet : isFinSet' A → isFinSet A
+isFinSet'→isFinSet (n , ∣ isFinSet-A ∣) = ∣ n , isFinSet-A ∣
+isFinSet'→isFinSet (n , squash p q i) = isPropIsFinSet (isFinSet'→isFinSet (n , p)) (isFinSet'→isFinSet (n , q)) i
+
+isFinSet≡isFinSet' : isFinSet A ≡ isFinSet' A
+isFinSet≡isFinSet' {A = A} = hPropExt isPropIsFinSet isPropIsFinSet' isFinSet→isFinSet' isFinSet'→isFinSet
 
-    from : FinSet' ℓ → FinSet ℓ
-    from (A , isFinSet'A) = A , transport (sym isFinSet≡isFinSet') isFinSet'A
+FinSet→FinSet' : FinSet ℓ → FinSet' ℓ
+FinSet→FinSet' (A , isFinSetA) = A , transport isFinSet≡isFinSet' isFinSetA
 
-    to-from : ∀ A → to (from A) ≡ A
-    to-from A = Σ≡Prop (λ _ → isPropIsFinSet') refl
+FinSet'→FinSet : FinSet' ℓ → FinSet ℓ
+FinSet'→FinSet (A , isFinSet'A) = A , transport (sym isFinSet≡isFinSet') isFinSet'A
```

Comment:
How about:

```
FinSet'→FinSet (A , isFinSet'A) = A , isFinSet'→isFinSet isFinSet'A
```

Reply:
haha now I got why you asked me to move these functions up.